component/aggregated_rbac.libsonnet

```diff
@@ -0,0 +1,103 @@
+local kube = import 'lib/kube.libjsonnet';
+
+local custom_resources = [
```

Comment:
Instead of keeping a list here, we could implement this as a postprocessing filter and create the list of custom resources based on the rendered set of CRDs deployed by the Helm chart. However, I think that's too complicated for the little benefit we get, as the set of CRDs for Rook seems fairly stable at this point.

Reply:
I had the same idea when I created the issue, but I agree that too much complexity for little gain